gigya_raas/gigya_raas.install

```diff
@@ -27,34 +27,49 @@ function gigya_raas_install() {
 	];
 
 	$schema = Database::getConnection()->schema();
+	$logger = \Drupal::logger('gigya_raas');
 
-	$error_message_format = sprintf('Unable to create column `%s` in the `%s` table during Gigya RaaS module installation.
-														If the column already exists from a previous installation of the module, you may ignore this warning. If the column already exists from another module, please contact Gigya\'s support.
-														If the column does not exist, please try to reinstall the module.');
+	$error_message_format = 'Unable to create column `%column` in the `%table` table during Gigya RaaS module installation. If the column already exists from a previous installation of the module, you may ignore this warning. If the column already exists from another module, please contact Gigya\'s support. If the column does not exist, please try to reinstall the module.';
```

Comment:
This change is required because logger messages are translatable and `\n` in texts makes it much harder